packaging/makeself/install-or-update.sh

```diff
@@ -252,23 +252,49 @@ replace_symlink() {
     ln -s "${target}" "${name}"
 }
 
+prep_ssl_dir() {
+  [ -L /opt/netdata/etc/ssl ] && rm /opt/netdata/etc/ssl
+  mkdir -p /opt/netdata/etc/ssl
+}
+
+select_cert_dir() {
+  prefix="/opt/netdata/etc/ssl/certs"
+
+  mkdir -p "${prefix}"
+
+  for bundle in ca-certificates.crt ca-bundle.crt; do
+    if [ -r "${1}/${bundle}" ]; then
+      replace_symlink "${1}/${bundle}" "${prefix}/ca-certificates.crt"
+      break
+    fi
+  done
+}
+
 select_system_certs() {
-  if [ -d /etc/pki/tls ] ; then
-    echo "${1} /etc/pki/tls for TLS configuration and certificates"
-    replace_symlink /etc/pki/tls /opt/netdata/etc/ssl
-  elif [ -d /etc/ssl ] ; then
-    echo "${1} /etc/ssl for TLS configuration and certificates"
-    replace_symlink /etc/ssl /opt/netdata/etc/ssl
-  fi
+  prep_ssl_dir
+
+  for dir in /etc/pki/tls /etc/ssl ; do
+    if [ -d "${dir}" ] ; then
+      echo "${1} ${dir} for TLS configuration and certificates"
+      select_cert_dir "${dir}/certs"
```

Comment:
This won't work on openSUSE because `ca-bundle.pem` is in `/etc/ssl/`, not `/etc/ssl/certs/`.